src/metatrain/pet/schema-hypers.json

```diff
@@ -0,0 +1,296 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
This whole schema file needs to be deleted. Hypers are now documented in python from `documentation.py`